vpr/src/pack/re_cluster.cpp

```diff
@@ -157,6 +165,8 @@ bool swap_two_molecules(t_pack_molecule* molecule_1,
                         bool during_packing,
                         int verbosity,
                         t_clustering_data& clustering_data) {
+    auto& cluster_ctx = g_vpr_ctx.mutable_clustering();
```

Comment:
Just curious, was there a reason the cluster ctx was defined up here, but used 30 lines below?

Reply:
Just wanted to be consistent with the coding style of other functions.